packages/core-toggle/readme.md

```diff
@@ -270,7 +261,7 @@ to create a component that behaves like a `<select>`:
 ```html
 <!--demo-->
 <div id="jsx-toggle-select"></div>
-<script type="text/jsx">
+<script type="text/javascript">
```

Comment:
Må ikke denne være jsx for at babel skal kicke inn? trenger kanskje ikke det? :)

Reply:
Ser ikke ut til å gjøre noen forskjell for stort annet enn syntax highlighting. (og at text/jsx ikke er en gyldig mime type). Babel prosesserer vel hele fila uansett?
VScode bruker textmate internt, som foreleøpig ikke ser ut til å bevege seg på tema: 
- https://github.com/microsoft/vscode/issues/145992
- https://github.com/textmate/html.tmbundle/issues/119

Reply:
True true, bare historikk som snakker her - var sånn i gamledager, kjør på :)